starknet_py/tests/e2e/docs/quickstart/test_using_account.py

```diff
@@ -45,8 +44,8 @@ async def test_using_account_client(new_account_client, map_compiled_contract):
     ]
```

Comment:
Can't review some of the lines, but there are mentions about AccountClient instead of Account in this file.